docs/dev_guides/changing_splink/contributing_to_docs.md

```diff
@@ -16,21 +16,27 @@ Once you've finished updating Splink documentation we ask that you run our spell
 
 ## Spellchecking docs
 
-When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_ and currently only works on systems which support `Homebrew` package manager. Instructions for other operating systems will be released later.
+When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_.
```

Comment:
In light of discussions on platform-compatibility and accessibility, I think it would be beneficial adding a sentence here to make sure people aren't put off from contributing even if they can't get the spellchecker to work.

```suggestion
When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. If for whatever reason you can't run the spellchecker on your system, please don't let this prevent you from contributing to the documentation. Please note, the spellchecker _only works on markdown files_.
```

Reply:
Good idea! Thanks.